### PR TITLE
Combines column and spanish class (and more)

### PR DESCRIPTION
**Pull request by BaoPun**
Able to switch in between games as well, albeit with a reset of the board and deck.

### CODE REVIEW

**Comment by phamkv9** [comment]
hey Bao. I liked how you made a little pun out of the branch name "game is done", maybe next time specify a bit more on which game function you worked on. It'd be easier for us all to know without having to look at the code changes. Thanks!

**Comment by BaoPun** [comment]
Yo scott, even though your pull request somehow got mixed in with my own, good job on creating both the decks for us!  It really helped me out on being able to implement both the decks into the game.  Maybe make the naming a little more clear for the spanish deck next time?

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 1]
```diff
@@ -194,10 +211,53 @@ <h1>Aces Up</h1>
     });
 });
 
+/*After clicking on this, clear the column data, make a new deck, and play with it now*/
+$("#reg_game").click(function(){
```
Comment (nguyvinc) [inline src/main/java/views/AcesUp/AcesUp.flt.html]: This is a neat solution to our problem, I never thought about being able to swap between decks while in game. The code seems to be free of bugs and looks good.

File: src/main/java/models/Deck.java [hunk 2]
```diff
@@ -9,28 +8,21 @@
...
 
-    public Deck(){
-        d = new ArrayList<>();
-        size = 0;
-        makeDeck();
-        shuffleDeck();
-    }
-
-    public void makeDeck(){
+    /*public void makeDeck(){
```
Comment (phamkv9) [inline src/main/java/models/Deck.java]: great job generating the deck, next time however maybe take this whole commented section out, it'll look better and doesn't confuse people to much

**Pull request merged**